var/spack/repos/builtin/packages/ape/package.py

```diff
@@ -6,40 +6,40 @@
 from spack import *
 
 
-class Ape(Package):
+class Ape(AutotoolsPackage):
     """A tool for generating atomic pseudopotentials within a Density-Functional
     Theory framework"""
 
     homepage = "http://www.tddft.org/programs/APE/"
-    url      = "http://www.tddft.org/programs/APE/sites/default/files/ape-2.2.1.tar.gz"
-
-    version('2.2.1', sha256='1bdb7f987fde81f8a5f335da6b59fa884e6d185d4a0995c90fde7c04376ce9e3')
-
+    url      = "https://gitlab.com/ape/ape/-/archive/2.3.2/ape-2.3.2.tar.gz"
```

Comment:
Is there any benefit of switching to Gitlab? The homepage still uses the previous URL, and those tarballs come with a configure script, so we don't need any Autotools dependencies.

Reply:
![image](https://user-images.githubusercontent.com/29532367/89780181-82aeb180-db43-11ea-8cb9-4a1fae0f0c0a.png)
I think gitlab is a more common url?
I don't know what versions we can use by access `download` link. But `Gitlab` can tell me that.

Reply:
I would argue that Download is a more common button that users press to download the package. Building directly from GitLab requires extra dependencies, which slows down install times.

Reply:
But `Download` Button only make you download the latest version.
That means I should check gitlab to find what versions they have, than try `spack checksum ape xxx xxx xxx` to get all packages from download link.
Is that the better way? 
I haven't added any other dependencies for changing this url, did I miss any dependency?

Reply:
If you hover over the link, you can see what version it will download based on the URL. Or you can download it and see the version from the tarball.

Changing the URL to Gitlab adds requirements for automake/autoconf/libtool/m4, which aren't required if you download from the old URL.